Wrong port in connectivity check calls in 3.0.7

**Describe the bug**
After upgrading to 3.0.7 `ReactiveNetwork.observeInternetConnectivity` stops working with:
```
E/ReactiveNetwork( 3672): Could not establish connection with WalledGardenStrategy
E/ReactiveNetwork( 3672): java.net.ConnectException: failed to connect to clients3.google.com/2607:f8b0:400b:809::200e (port 80) after 2000ms: isConnected failed: ENETUNREACH (Network is unreachable)
E/ReactiveNetwork( 3672): 	at libcore.io.IoBridge.isConnected(IoBridge.java:238)
E/ReactiveNetwork( 3672): 	at libcore.io.IoBridge.connectErrno(IoBridge.java:171)
E/ReactiveNetwork( 3672): 	at libcore.io.IoBridge.connect(IoBridge.java:122)
E/ReactiveNetwork( 3672): 	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:183)
E/ReactiveNetwork( 3672): 	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:456)
E/ReactiveNetwork( 3672): 	at java.net.Socket.connect(Socket.java:882)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.Platform.connectSocket(Platform.java:174)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.Connection.connect(Connection.java:152)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:276)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:211)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:382)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:332)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.HttpURLConnectionImpl.getResponseCode(HttpURLConnectionImpl.java:500)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.DelegatingHttpsURLConnection.getResponseCode(DelegatingHttpsURLConnection.java:105)
E/ReactiveNetwork( 3672): 	at com.android.okhttp.internal.http.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:25)
E/ReactiveNetwork( 3672): 	at com.github.pwittchen.reactivenetwork.library.rx2.internet.observing.strategy.WalledGardenInternetObservingStrategy.isConnected(WalledGardenInternetObservingStrategy.java:109)
E/ReactiveNetwork( 3672): 	at com.github.pwittchen.reactivenetwork.library.rx2.internet.observing.strategy.WalledGardenInternetObservingStrategy$1.apply(WalledGardenInternetObservingStrategy.java:66)
E/ReactiveNetwork( 3672): 	at com.github.pwittchen.reactivenetwork.library.rx2.internet.observing.strategy.WalledGardenInternetObservingStrategy$1.apply(WalledGardenInternetObservingStrategy.java:64)
E/ReactiveNetwork( 3672): 	at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:57)
E/ReactiveNetwork( 3672): 	at io.reactivex.internal.operators.observable.ObservableInterval$IntervalObserver.run(ObservableInterval.java:82)
E/ReactiveNetwork( 3672): 	at io.reactivex.Scheduler$PeriodicDirectTask.run(Scheduler.java:532)
E/ReactiveNetwork( 3672): 	at io.reactivex.Scheduler$Worker$PeriodicTask.run(Scheduler.java:479)
E/ReactiveNetwork( 3672): 	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
E/ReactiveNetwork( 3672): 	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
E/ReactiveNetwork( 3672): 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
E/ReactiveNetwork( 3672): 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
E/ReactiveNetwork( 3672): 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
E/ReactiveNetwork( 3672): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
E/ReactiveNetwork( 3672): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
E/ReactiveNetwork( 3672): 	at java.lang.Thread.run(Thread.java:818)
E/ReactiveNetwork( 3672): Caused by: android.system.ErrnoException: isConnected failed: ENETUNREACH (Network is unreachable)
E/ReactiveNetwork( 3672): 	at libcore.io.IoBridge.isConnected(IoBridge.java:223)
E/ReactiveNetwork( 3672): 	... 29 more
```

**To Reproduce**
Call `ReactiveNetwork.observeInternetConnectivity`.

**Expected behavior**
`true` is emitted on detected network connectivity.

**Smartphone (please complete the following information):**
 - Device: Pixel 2 (emulator)
 - OS: Android 10
 - Library Version: 3.0.7

**Additional context**
3.0.6 works as expected with the default config and allowed cleartext.

The default config with 3.0.7 tries to reach `https://clients3.google.com/generate_204` on port `80`.

The following version with the manual port works as expected:
```
ReactiveNetwork.observeInternetConnectivity(
    InternetObservingSettings.builder().host("https://clients3.google.com/generate_204").port(443).build()
)!!
```

--- a/build.gradle
+++ b/build.gradle
@@ -3,12 +3,12 @@
 apply plugin: 'io.codearte.nexus-staging'
 
 ext {
-  minSdkVersion = 9
+  minSdkVersion = 14
   minSdkVersionApps = 23
   compileSdkVersion = 28
   buildToolsVersion = '28.0.3'
   gradleVersion = '4.6'
-  kotlinVersion = '1.3.70'
+  kotlinVersion = '1.3.71'
   detektVersion = '1.0.0.RC6-1'
 }
 

--- a/gradle.properties
+++ b/gradle.properties
@@ -19,3 +19,4 @@ org.gradle.jvmargs=-XX:MaxPermSize=1024m -XX:+CMSClassUnloadingEnabled -XX:+Heap
 android.useAndroidX=true
 android.enableJetifier=true
 android.enableR8.fullMode=false
+android.enableUnitTestBinaryResources=true

--- a/library/build.gradle
+++ b/library/build.gradle
@@ -47,6 +47,8 @@ android {
     }
   }
 
+  testOptions.unitTests.includeAndroidResources = true
+
   jacocoAndroidUnitTestReport {
     csv.enabled false
     html.enabled true

--- a/library/src/main/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/InternetObservingSettings.java
+++ b/library/src/main/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/InternetObservingSettings.java
@@ -131,7 +131,7 @@ public final class InternetObservingSettings {
   public final static class Builder {
     private int initialInterval = 0;
     private int interval = 2000;
-    private String host = "https://clients3.google.com/generate_204";
+    private String host = "http://clients3.google.com/generate_204";
     private int port = 80;
     private int timeout = 2000;
     private int httpResponse = HttpURLConnection.HTTP_NO_CONTENT;

--- a/library/src/main/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/strategy/WalledGardenInternetObservingStrategy.java
+++ b/library/src/main/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/strategy/WalledGardenInternetObservingStrategy.java
@@ -40,7 +40,7 @@ import javax.net.ssl.HttpsURLConnection;
  * if a device is connected to the Internet or not.
  */
 @Open public class WalledGardenInternetObservingStrategy implements InternetObservingStrategy {
-  private static final String DEFAULT_HOST = "https://clients3.google.com/generate_204";
+  private static final String DEFAULT_HOST = "http://clients3.google.com/generate_204";
   private static final String HTTP_PROTOCOL = "http://";
   private static final String HTTPS_PROTOCOL = "https://";
 

--- a/library/src/test/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/InternetObservingSettingsTest.java
+++ b/library/src/test/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/InternetObservingSettingsTest.java
@@ -46,7 +46,7 @@ public class InternetObservingSettingsTest {
     // then
     assertThat(settings.initialInterval()).isEqualTo(0);
     assertThat(settings.interval()).isEqualTo(2000);
-    assertThat(settings.host()).isEqualTo("https://clients3.google.com/generate_204");
+    assertThat(settings.host()).isEqualTo("http://clients3.google.com/generate_204");
     assertThat(settings.port()).isEqualTo(80);
     assertThat(settings.timeout()).isEqualTo(2000);
     assertThat(settings.httpResponse()).isEqualTo(204);

--- a/library/src/test/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/strategy/WalledGardenInternetObservingStrategyTest.java
+++ b/library/src/test/java/com/github/pwittchen/reactivenetwork/library/rx2/internet/observing/strategy/WalledGardenInternetObservingStrategyTest.java
@@ -160,7 +160,7 @@ import static org.mockito.Mockito.when;
 
     // when
     HttpURLConnection connection =
-        strategy.createHttpsUrlConnection(getHost(), PORT, TIMEOUT_IN_MS);
+        strategy.createHttpsUrlConnection("https://clients3.google.com", PORT, TIMEOUT_IN_MS);
 
     // then
     assertThat(connection).isNotNull();
@@ -176,7 +176,7 @@ import static org.mockito.Mockito.when;
     // given
     final String errorMsg = "Could not establish connection with WalledGardenStrategy";
     final IOException givenException = new IOException(errorMsg);
-    when(strategy.createHttpsUrlConnection(getHost(), PORT, TIMEOUT_IN_MS)).thenThrow(
+    when(strategy.createHttpUrlConnection(getHost(), PORT, TIMEOUT_IN_MS)).thenThrow(
         givenException);
 
     // when

